sql_metadb/derived_tables/runlist.txt

```diff
@@ -39,6 +39,7 @@ instance_tags.sql
 invoice_adjustments_in_addition_to.sql
 invoice_adjustments_ext.sql
 invoice_lines_adjustments.sql
+invoice_lines_fund_distributions.sql
```

Comment:
@sbeltaine Here again, invoice_lines_fund_distributions.sql is already present in the runlist.